### PR TITLE
Added Display Command

### PR DESCRIPTION
**Pull request by dubeyPraY**
Added display command to print untranspiled circuits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,9 +203,10 @@ options:
                         Comma-separated list of hex values for character(s) to emit at the head of each line of job monitor output, default is '0x0d'
   --job_monitor_filepath JOB_MONITOR_FILEPATH
                         Filepath for Job Monitor output if Job Monitor requested by --use_job_monitor, default is sys.stdout
-  -w, --warnings        Don't print warnings on missing optional modules
```
Comment (jwoehr) [inline README.md:212]: I think you dropped the word "modules" from the description here.

**Comment by jwoehr** [comment]
Another thought: the workflow of making pull requests works better if your own fork of QisJob keeps its `main` branch sync'ed with QisJob `main`. To do your development, you should branch first, push that branch to your fork on GitHub, and create your pull request from the branch instead of from your fork's main.

**Comment by jwoehr** [comment]
Otherwise, nice work, @dubeyPraY ! Thank you.
Fix the description of that one command-line option and I'll merge this.

**Comment by dubeyPraY** [comment]
Sir, I have updated the readme file.

**Pull request merged**